### ISSUE TITLE
Try to scale down Prometheus and alertmanager replica

### ISSUE THREAD

**Issue by praveenkumar** [code-ready/snc#350]
Right now when user enable monitoring then the alertmanager and Prometheus statefulset have 3 and 2 replica sets, which doesn't make sense for single node cluster, try to check if we can able to set those replica to 1 without CVO overrides.

```
NAME                                 READY   AGE
statefulset.apps/alertmanager-main   3/3     3m14s
statefulset.apps/prometheus-k8s      1/2     3m13s
```

**Comment by praveenkumar** [comment]
https://github.com/openshift/cluster-monitoring-operator/pull/1077 something automatic happen on 4.8 time frame.

**Comment by praveenkumar** [comment]
https://github.com/openshift/cluster-monitoring-operator/pull/1077 is merged and with 4.8 there will be only single replica for prometheus and alertmanager.
